extension/js/common/core/pgp-key.ts

```diff
@@ -308,4 +319,9 @@ export class PgpKey {
     }
     throw new Error('No valid signature found in key');
   }
+
+  static cryptoKeyOptionalMatchingKeyid = (key: OpenPGP.key.Key, forMsgKeyids: OpenPGP.Keyid[] | undefined) => {
```

Comment:
Actually, despite the name, this one should go into `PgpMsg` because I believe it will be used there privately.

Reply:
Okay, I will check it